_test/test_mpi_wrappers/test_cluster_wrapper.m

```diff
@@ -314,7 +314,9 @@ function test_init_fails_if_n_workers_gt_num_logical_cores(~)
                 'HERBERT:ClusterParpoolWrapper:invalid_argument');
             clob3 = onCleanup(@()finalize_all(clust));
         end
-
+    end
+    % keep userpath
```

Comment:
completely unclear why this comment is here

Reply:
because it explains what the code below is doing.